packages/playwright/src/reporter.ts

```diff
@@ -373,13 +373,13 @@ export default class ReplayPlaywrightReporter implements Reporter {
       const output: string[] = [];
 
       if (!didUseReplayBrowser) {
-        mixpanelAPI.trackEvent("playwright.warning.reporter-used-without-replay-project");
+        mixpanelAPI.trackEvent("warning.reporter-used-without-replay-project");
         output.push(emphasize("None of the configured projects ran using Replay Chromium."));
       }
 
       if (!isReplayBrowserInstalled) {
         if (didUseReplayBrowser) {
-          mixpanelAPI.trackEvent("playwright.warning.replay-browser-not-installed");
+          mixpanelAPI.trackEvent("warning.replay-browser-not-installed");
```

Comment:
This isn't strictly related to the change but I noticed we're prepending an unnecessary "playwright." here. (The events will all already get prefixed with "@replayio/playwright.")